.dockerignore

```diff
@@ -7,8 +7,6 @@ screenshots
 tools
 test/data/registry
 venv
-.git
-!.git/HEAD
```

Comment:
This will result in the `.git` directory being in the image, which is a no-go.

Reply:
@josephschorr no, i remove in the Dockerfile (i'll update Dockerfile.rhel7, etc. when we're agreed on this PR)